src/semeio/fmudesign/__init__.py

```diff
@@ -0,0 +1,19 @@
+"""
+Used for pre and processing of ert sensitivities, such as
+setting up design matrix to run single sensitivities with ERT and
+post processing of results to create input for a tornado plot.
+Output of this module can be used in custom standalone applications.
+"""
+
+from semeio.fmudesign._designsummary import summarize_design
+from semeio.fmudesign._excel2dict import excel2dict_design, inputdict_to_yaml
+from semeio.fmudesign._tornado_onebyone import calc_tornadoinput
+from semeio.fmudesign.create_design import DesignMatrix
+
+__all__ = [
+    "summarize_design",
+    "calc_tornadoinput",
+    "DesignMatrix",
+    "excel2dict_design",
+    "inputdict_to_yaml",
+]
```

Comment:
I think overall the changes are good, my only worry is that people will have to change to `from semeio.fmudesign import ...` if they are importing this functionality in their scripts directly. Do we have any idea if people are doing this at all, and if yes to what extent? :thinking:

Reply:
@tralsos do you have any comments here?

Reply:
See this discussion: https://github.com/equinor/fmu-tools/pull/246#pullrequestreview-2413109007